code/modules/clothing/glasses/glasses.dm

```diff
@@ -10,7 +10,7 @@
 	var/prescription = 0
 	var/prescription_upgradable = 0
 	var/see_darkness = 1
-	var/HUDType = 0
+	var/HUDType = null

```

Comment:
The value was on the glasses before just changed itasf.  To null can move t if it's okay